src/operators/LazyTable.cpp

```diff
@@ -0,0 +1,92 @@
+//
+// Created by Nicholas Corrado on 4/5/20.
+//
+
+#include "OperatorResult.h"
+
+#include <utility>
+#include <iostream>
+#include "table/util.h"
+#include "LazyTable.h"
+
+namespace hustle::operators{
+
+    LazyTable::LazyTable() {
```

Comment:
I'm not sure I fully understand the concept of a `LazyTable`, and as noted above a quick round of comments even in this round would be really sweet!